lib/ecto/schema.ex

```diff
@@ -195,6 +198,7 @@ defmodule Ecto.Schema do
       import Ecto.Schema, only: [schema: 2]
       @primary_key {:id, :integer, read_after_writes: true}
       @timestamps_type Ecto.DateTime
+      @timestamps_use_usec true
```

Comment:
The default should be false (i.e. we should default to the behaviour that works out of the box on all adapters).